### DISCUSSION TITLE
How to use types like chrono<Utc> "DateTime" in a dynamic schema?

### DISCUSSION THREAD

**Discussion by mathroc** [async-graphql/async-graphql#1208] [Q&A] [answered]
I'm trying my hand at building dynamic schema, I think it's going to be awesome :) And I'm wondering how to declare a `DateTime` field, I have the "chrono" feature enabled but how do I specify the field type

```rust
    let query =
        Object::new("Query").field(Field::new(
            "now",
            TypeRef::named_nn(TypeRef::INT), // What should I put instead of this INT declaration here?
            |_| {
                FieldFuture::new(async {
                    Ok(Some(async_graphql::Value::from(chrono::Utc::now()))) // and should this work?
                })
            },
        ));
```

thank you!

**Comment by sunli829** [comment, marked as answer]
You should create a custom scalar, like this

https://github.com/async-graphql/async-graphql/blob/ee8b57bb3cbef2cd778418d0bd5d18572ba38608/src/dynamic/scalar.rs#L140
